### PR TITLE
Add korean-date-converter extension

### PR DESCRIPTION
**Pull request by youngkyo0504**
## Description

This PR introduces a new Raycast extension called "Korean Date Converter". This extension allows users to convert Korean date expressions (such as "다음주 화요일", "3일 후") into actual dates.(`10/08` or `10/11`) 

It provides a simple and efficient way for users to interpret relative date expressions in Korean within the Raycast environment.

## Screencast

image

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/korean-date-converter/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Korean Date Converter
+
+한국어로 된 상대적인 날짜와 시간 표현을 날짜로 변경해줍니다. 이 라이브러리를 사용하면 "다음주 화요일", "3일 후",  등의 다양한 한국어 표현을 날짜로 쉽게 변환할 수 있습니다.
```
Comment (pernielsentikaer) [inline extensions/korean-date-converter/README.md:3]: Could you add this in English instead? You can create a README-ko.md if needed with this.

Comment (youngkyo0504) [reply extensions/korean-date-converter/README.md]: Thank you. I worked on it. It does not need README-ko.md.

**Comment by youngkyo0504** [comment]
> What about a preference to let the user choose between copy/paste?

I add commit which implement preference. https://github.com/raycast/extensions/pull/14989/commits/df544d930cca5385fe2e9f69b3ea90cb2018f065

**Review by youngkyo0504 — COMMENTED** [review]
I add commit which implement preference. https://github.com/raycast/extensions/commit/df544d930cca5385fe2e9f69b3ea90cb2018f065

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**